src/components/viewer.js

```diff
@@ -35,10 +35,16 @@ const isDangerousUrl = (url) => {
   return !whitelistedProtocols.some(proto => url.startsWith(proto));
 };
 
+const containsJS = (markup) => markup.indexOf('javascript&colon;') !== -1;
```

Comment:
what about testing for the existence of `'javascript:'`? does `String#indexOf` parse entities?